gov_uk_dashboards/components/plotly/tooltip.py

```diff
@@ -0,0 +1,32 @@
+"""tooltip"""
+from dash import html
+import dash_bootstrap_components as dbc
+
+
+def tooltip(tooltip_text: str, tooltip_id: str, color: str, font_size: int):
+    """Creates a tooltip for providing further information
+
+    Args:
+        tooltip_text (str): The text to display when hovering
+        tooltip_id (str): The element ID to contain the tooltip text
+        color (str): The color of the tooltip text
+        font_size (int): The size of the tooltip text
+
+    Returns:
+        html.Span: The dash HTML object for the tooltip.
+    """
+
+    return html.Span(
+        [
+            " ⓘ",
+            dbc.Tooltip(
+                tooltip_text,
+                target=tooltip_id,
+                class_name="tooltip tooltiptext",
+                style={"color": color, "font-size": font_size},
```

Comment:
Repeating a comment from the other PR as relevant here too - I am not sure how this will be interpreted by the CSS - font sizes are usually specified with a measurement (e.g. px or em) - do we know what this will result in? Gov Design System (and other sources on accessibility I've seen) suggest a minimum size of 16px, which is equal to 12em, so this might be okay? But might be safer to specify "16px"

Reply:
Thank you.